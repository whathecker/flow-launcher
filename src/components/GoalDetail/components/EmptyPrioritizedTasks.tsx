import React from "react";
import { Image, StyleSheet } from "react-native";
import { View, Text } from "../../Themed";
import { Container, Color, Typography } from "../../../styles";

const EmptyPrioritizedTasks: React.FC = () => {
  return (
    <View style={styles.emptyTasksMsgWrapper}>
      <Image
        style={styles.emptyTasksImage}
        source={require(`../../../../assets/images/partying-face_1f973.png`)}
      />
      <Text style={styles.emptyTasksMsg}>
        {"Small steps, great achievement"}
      </Text>
    </View>
  );
};

const styles = StyleSheet.create({
  emptyTasksMsgWrapper: {
    ...Container.centerAlignedVertical,
    height: 280,
    padding: 20,
  },
  emptyTasksImage: {
    width: 40,
    height: 40,
  },
  emptyTasksMsg: {
    ...Typography.p,
    fontSize: 14,
    color: Color.light.subtleLabel,
    paddingTop: 20,
  },
});

export default EmptyPrioritizedTasks;
