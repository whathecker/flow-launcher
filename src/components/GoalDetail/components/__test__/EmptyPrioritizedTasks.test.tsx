import * as React from "react";
import renderer from "react-test-renderer";
import EmptyPrioritizedTasks from "../EmptyPrioritizedTasks";

describe("Test EmptyPrioritizedTasks component", () => {
  it("Should renders EmptyPrioritizedTasks correctly", () => {
    const tree = renderer.create(<EmptyPrioritizedTasks />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
