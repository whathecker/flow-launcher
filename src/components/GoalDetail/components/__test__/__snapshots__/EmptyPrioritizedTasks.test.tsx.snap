// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test EmptyPrioritizedTasks component Should renders EmptyPrioritizedTasks correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
      },
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "height": "80%",
        "justifyContent": "center",
        "padding": 20,
      },
    ]
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  />
  <Text
    style={
      Array [
        Object {
          "color": "#000000",
        },
        Object {
          "color": "#848484",
          "fontFamily": "Forum-Regular",
          "fontSize": 16,
          "paddingTop": 20,
        },
      ]
    }
  >
    Small steps, great achievement
  </Text>
</View>
`;
